src/test/java/com/artipie/asto/CopyTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.asto;
+
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.memory.InMemoryStorage;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * A test for {@link Copy}.
+ * @since 0.19
+ * @checkstyle LocalFinalVariableNameCheck (500 lines)
+ * @checkstyle StringLiteralsConcatenationCheck (500 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public class CopyTest {
+
+    @Test
+    public void copyTwoFilesFromOneStorageToAnotherWorksFine()
```

Comment:
@Sammers21 why two files not one or three? Maybe we can name this method like `copyFilesFromOneStorageToAnother`?

Reply:
@g4s8, @Vatavuk asked me to be more descriptive and the name completely reflects the actions inside. What is wrong with it?

Reply:
@Sammers21 he asked it because it was `copyWorks`, now it contains redundant implementations details. The method name should be named in a such way: if you see build logs `Error: test <test name> failed`, you should understand what exactly was failed. If you see `copyTwoFilesFromOneStorageToAnotherWorksFine failed` you may be confused why two files can't be copied from one storage to another, did you get the point?